### DISCUSSION TITLE
Owner Minting Specific Token ID

### DISCUSSION THREAD

**Discussion by tomvanriet** [TENK-DAO/tenk#93] [General]
Hey Willem, 

Just wanted to find out if its possible for the contract owner to mint a specific token ID (for giveaways, ect) without upsetting the raffle?

The only alternative to that is to mint first then change the URI?

Kind Regards,
Tom

**Comment by willemneal** [comment]
Hey Tom.  Currently the easiest way to do this is to separate your tokens beforehand. For example, `0-X` will be for the raffle and `X+1...` will be for the giveaways.  This way you have the raffle as normal with size `X` and then have a separate method to mint as the owner for the giveaways.

It is possible if you know which before the raffle starts to force the raffle to pick a certain token id, but requires extra care.
